converter/rst2markdown.py

```diff
@@ -245,15 +244,14 @@ def _code_lines(self, data):
         return "\n".join(lines)
 
     def _extrtoolembed(self, matchobj):
-        caret_token = self._caret_token
         name = matchobj.group('name').lower()
         ex_data = self._exercises.get(name.lower(), {})
         assessment_id = f'test-{name.lower()}'
         if not ex_data:
-            return f'{caret_token}***Exercise:*** "{name}" **not found**{caret_token}{caret_token}'
+            return
```

Comment:
should it return an empty string?

Reply:
Yes. To avoid displaying exercise titles on the page where they are missing

Reply:
What is the difference with line 254? Not sure if I understand it

Reply:
fixed it